src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialWikiActivityPageObject.java

```diff
@@ -89,6 +95,20 @@ public void verifyRecentEdition(String articleName, String userName) {
   }
 
   /**
+   * verifies if given edition on WikiActivity is not present, searching through 5 recent editions.
+   */
+//  public void verifyNoRecentEditionIsPresent(String articleName, String userName) {
+//
+//      List<Activity> activityList = getActivityList();
+//      Integer count = activityList.filter(p->p.getTitle().equals("test")).count();
+//      Assertion.assertEquals(count.toString(), "0");
+//  }
+
+//  public List<Activity> getActivityList(){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)